lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerLaunchTests.cs

```diff
@@ -98,7 +98,7 @@ public class PuppeteerLaunchTests
             Assert.True(Directory.GetFiles(userDataDir).Length > 0);
             await browser.CloseAsync();
             Assert.True(Directory.GetFiles(userDataDir).Length > 0);
-            // Directory.Delete(userDataDir, true);
+            Directory.Delete(userDataDir, true);
```

Comment:
adding
```cs
await Task.Delay(1000);
```
right before this line seems to work, not sure what's not waiting

Reply:
yeah... I don't like much that solution. Another solution I read is looping (and failing) until the file is released https://stackoverflow.com/a/37154588/2373249.

I don't care about that delete. The problem is that `Launcher.cs` is also failing to delete the folder here https://github.com/kblok/puppeteer-sharp/blob/b99216161da585f74a1981b466cece9db42ed32b/lib/PuppeteerSharp/Launcher.cs#L250 and we do need to prevent that from happening.